tools/system_libs.py

```diff
@@ -1082,6 +1082,10 @@ def __init__(self, **kwargs):
     self.is_webgl2 = kwargs.pop('is_webgl2')
     self.is_ofb = kwargs.pop('is_ofb')
     self.is_full_es3 = kwargs.pop('is_full_es3')
+    if self.is_webgl2 or self.is_full_es3:
+      # Don't use append or += here, otherwise we end up adding to
+      # the class member.
+      self.src_files = self.src_files + ['webgl2.c']
```

Comment:
What does this mean? Is this not the same as
```py
self.src_files += ['webgl2.c']
```
?

Reply:
+= is like append/extent.  It modifies the array inplace.   In this case we explicitly don't want to do that because the array is a class member.